workflows/fetchdatairidanext.nf

```diff
@@ -62,11 +63,19 @@ workflow FETCHDATAIRIDANEXT {
             return [["id": new_id, "irida_id": meta.irida_id[0], "insdc_accession": meta.insdc_accession[0]], meta.insdc_accession[0]]
         }
 
-    FASTQ_DOWNLOAD_PREFETCH_FASTERQDUMP_SRATOOLS (
-        ch_sra_ids = input,
-        ch_dbgap_key = []
-    )
-    ch_versions = ch_versions.mix(FASTQ_DOWNLOAD_PREFETCH_FASTERQDUMP_SRATOOLS.out.versions)
+    // Determine workflow based off of the provider
+    if ( params.provider.toLowerCase() == 'sra' ) {
```

Comment:
My preference would be having the `sra` part here as a constant defined earlier.